example/image-classification/predict-cpp/image-classification-predict.cc

```diff
@@ -32,252 +48,254 @@ const mx_float DEFAULT_MEAN = 117.0;
+  // Read all kinds of file into a BGR color 3 channels image
+  cv::Mat im_ori = cv::imread(image_file, cv::IMREAD_COLOR);
 
-    if (im_ori.empty()) {
-        std::cerr << "Can't open the image. Please check " << image_file << ". \n";
-        assert(false);
-    }
-
-    cv::Mat im;
+  if (im_ori.empty()) {
+    std::cerr << "Can't open the image. Please check " << image_file << ". \n";
+    assert(false);
+  }
 
-    resize(im_ori, im, resize_size);
+  cv::Mat im;
 
-    int size = im.rows * im.cols * channels;
+  resize(im_ori, im, resize_size);
 
-    mx_float* ptr_image_r = image_data;
-    mx_float* ptr_image_g = image_data + size / 3;
-    mx_float* ptr_image_b = image_data + size / 3 * 2;
+  int size = im.rows * im.cols * channels;
 
-    float mean_b, mean_g, mean_r;
-    mean_b = mean_g = mean_r = DEFAULT_MEAN;
+  mx_float* ptr_image_r = image_data;
+  mx_float* ptr_image_g = image_data + size / 3;
+  mx_float* ptr_image_b = image_data + size / 3 * 2;
 
-    for (int i = 0; i < im.rows; i++) {
-        uchar* data = im.ptr<uchar>(i);
+  float mean_b, mean_g, mean_r;
+  mean_b = mean_g = mean_r = DEFAULT_MEAN;
 
-        for (int j = 0; j < im.cols; j++) {
-            if (mean_data) {
-                mean_r = *mean_data;
-                if (channels > 1) {
-                    mean_g = *(mean_data + size / 3);
-                    mean_b = *(mean_data + size / 3 * 2);
-                }
-               mean_data++;
-            }
-            if (channels > 1) {
-                *ptr_image_b++ = static_cast<mx_float>(*data++) - mean_b;
-                *ptr_image_g++ = static_cast<mx_float>(*data++) - mean_g;
-            }
+  for (int i = 0; i < im.rows; i++) {
+    uchar* data = im.ptr<uchar>(i);
 
-            *ptr_image_r++ = static_cast<mx_float>(*data++) - mean_r;;
+    for (int j = 0; j < im.cols; j++) {
+      if (mean_data) {
+        mean_r = *mean_data;
+        if (channels > 1) {
+          mean_g = *(mean_data + size / 3);
+          mean_b = *(mean_data + size / 3 * 2);
         }
+        mean_data++;
+      }
+      if (channels > 1) {
+        *ptr_image_b++ = static_cast<mx_float>(*data++) - mean_b;
+        *ptr_image_g++ = static_cast<mx_float>(*data++) - mean_g;
+      }
+
+      *ptr_image_r++ = static_cast<mx_float>(*data++) - mean_r;;
     }
+  }
 }
 
 // LoadSynsets
 // Code from : https://github.com/pertusa/mxnet_predict_cc/blob/master/mxnet_predict.cc
-std::vector<std::string> LoadSynset(std::string synset_file) {
-    std::ifstream fi(synset_file.c_str());
+std::vector <std::string> LoadSynset(std::string synset_file) {
+  std::ifstream fi(synset_file.c_str());
 
-    if ( !fi.is_open() ) {
-        std::cerr << "Error opening synset file " << synset_file << std::endl;
-        assert(false);
-    }
+  if (!fi.is_open()) {
+    std::cerr << "Error opening synset file " << synset_file << std::endl;
+    assert(false);
+  }
 
-    std::vector<std::string> output;
+  std::vector <std::string> output;
```

Comment:
is this your IDE settings?

Reply:
Thanks, this was a weird default setting on a new machine